build-now.lua

```diff
@@ -435,9 +435,18 @@ local function build_construction(bld)
     local construction_type = bld.type
     dfhack.buildings.deconstruct(bld)
 
+    -- check for and remove a construction on the tile already (i.e. building a construction on top of a wall)
+    local use_open_space_for_original_tile = true
```

Comment:
this should only be true if the original tile is a construction

Reply:
Oops. I meant to initialise that as false and set it to true in the loop.